### PR TITLE
chore: update node engine

### PR DESCRIPTION
**Pull request by snitin315**
Update node

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Webpack loader to automatically npm install & save dependencies.",
   "main": "index.js",
   "engines": {
-    "node": ">=4.2.0"
+    "node": ">=10.13.0"
```
Comment (alexander-akait) [inline package.json:7]: Let's use `12` LTS

Comment (snitin315) [reply package.json:7]:
```suggestion
    "node": ">=12.13.0"
```

**Comment by anshumanv** [comment]
updated here https://github.com/webpack-contrib/npm-install-webpack-plugin/pull/149#discussion_r619670480

**Comment by snitin315** [comment]
Fixed in master

**Pull request closed without merging**